nilearn/tests/test_niimg.py

```diff
@@ -77,3 +78,11 @@ def test_img_data_dtype():
     # Verify that the distinction is worth making
     assert any(dtype_matches)
     assert not all(dtype_matches)
+
+
+def test_load_niimg():
+    img = Nifti1Image(np.zeros((10, 10, 10)), np.eye(4))
```

Comment:
It seems like several tests of this file use this kind of toy data. I think we could start factorizing by writing a fixture.
It will make things easier when further factorizing things across Nilearn in a `data_generation` module as we discussed some time ago with @jeromedockes 
WDYT?